cmd/clusterctl/config/assets/cert-manager.yaml

```diff
@@ -1,8 +1,10 @@
 # This is manually edited cert-manager v0.11.0 release.
-# We've manually removed the ClusterRoleBinding cert-manager-leaderelection.
-# OTHER NOTES:
+# DIFFERENCES:
+# 1. We've manually removed the ClusterRoleBinding cert-manager-leaderelection.
 # See https://github.com/kubernetes-sigs/cluster-api/issues/2928
 # See https://github.com/jetstack/cert-manager/pull/2207
+# 2. Added `sideEffects: None` to MutatingWebhookConfiguration.
+# See https://github.com/kubernetes-sigs/cluster-api/issues/3204
```

Comment:
Now I kinda wish we used kustomize to apply these patches, so they can be reproducible, oh well :)

Reply:
Yeah...I agree. The patches would've also acted as the documentation instead of having these comments.

Reply:
@vincepri Did you want me to create an issue for generating this asset file via kustomize?

Reply:
Yeah, not a huge problem though for now